### PR TITLE
[AKS]Support associating an existing user assigned identity for MSI cluster

### PR DESCRIPTION
**Pull request by norshtein**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/index.json [hunk 1]
```diff
@@ -1188,6 +1188,38 @@
                     "version": "0.4.55"
                 },
                 "sha256Digest": "d6fa0ec6dfffa82941d9b93415b317f5aee42191c3f6193fbc5842f36822cc48"
+            },
+            {
+                "downloadUrl": "https://azurecliaks.blob.core.windows.net/azure-cli-extension/aks_preview-0.4.56-py2.py3-none-any.whl",
```
Comment (norshtein) [inline src/index.json:1193]: I just found in latest version aks-preview, the wheel file is stored in storage account `azurecliprod`, how can I upload the extension to it?

Comment (norshtein) [reply src/index.json]: Just realized the blob and index will be updated by CI, reverted the index change.

**Comment by yonzhan** [comment]
AKS

**Comment by yungezz** [comment]
hi @norshtein could  you pls resolve conflict for next step?

**Comment by norshtein** [comment]
Hi @arrownj , @yungezz , I've fixed the merge conflict, can we merge the PR?

**Pull request merged**